README.md

```diff
@@ -71,3 +71,37 @@ Update the version number in the local package.json file so that it interacts co
 
 ### Inputs
 - `version` - The new active version number of the package
+
+## [build-with-ubp](https://github.com/mindjolt/uc-actions/tree/main/build-with-ubp)
+
+Create a build using UBP, supports both UnityCore version and UC-UBP-SDK versions.
+
+### Inputs
+- `files` - The files you want to build with space delimited
+    - Files with `%PROJECT_PATH%` will have the UBP Saved Project settings path inserted
+    - Files that start with `./` will use what ever directory was last used.
+- `platform` - The UBP platform you with to build.
+- `unityVersion` - The version of Unity to build with.
+- `override` - The JSON string to use for the override file.
+  - To create the JSON it is recommended to use [object-revamp](https://github.com/nickofthyme/object-remap)
+- <i>(optional)</i> `buildWithUnityCore` If set to `true` will target UnityCore UBP instead
```

Comment:
Instead of this very specific boolean, it might be nice to just have a `method` input that defaults to `JamCity.UnityCore.UnifiedBuildPipelineSdk.Editor.Commands.BuildPlayerFromFile` but could be overridden if desired.  That would make this more flexible and also make the actual action simpler.